js/serial_backend.js

```diff
@@ -181,28 +182,31 @@ function onOpen(openInfo) {
         }, 10000);
 
         FC.resetState();
-
+        MSP.listen(update_packet_error);
+        mspHelper = new MspHelper();
+        MSP.listen(mspHelper.process_data);
```

Comment:
You need to bind() the `process_data` function here, or `this` references inside it will not work:

`MSP.listen(mspHelper.process_data.bind(mspHelper))`